shared/common-adapters/confirm-modal/index.js

```diff
@@ -0,0 +1,126 @@
+// @flow
+import * as React from 'react'
+import {Box, Box2, ButtonBar, HeaderOrPopup, Icon, Text, WaitingButton} from '..'
+import * as Styles from '../../styles'
+import type {IconType} from '../icon.constants'
+
+// generally one of icon or header will be given
+export type Props = {|
+  confirmText?: string,
+  content?: React.Node,
+  description: string,
+  header?: React.Node,
+  icon?: IconType,
+  onCancel: () => void,
+  onConfirm: () => void,
+  prompt: string,
+  waitingKey?: string,
+|}
+
+class _ConfirmModal extends React.Component<Props> {
+  render() {
+    return (
+      <Box style={styles.mobileFlex}>
+        <Box2 direction="vertical" style={styles.container}>
+          <Box2
+            alignItems="center"
+            direction="vertical"
+            fullWidth={true}
+            fullHeight={true}
+            style={styles.inner}
+          >
+            {this.props.icon && (
+              <Icon
+                boxStyle={styles.icon}
+                color={Styles.globalColors.black_50}
+                fontSize={Styles.isMobile ? 64 : 48}
```

Comment:
There's a `sizeType` now: https://github.com/keybase/client/blob/9de8767f5dec74c2171490fa4301b6f36469eadc/shared/common-adapters/icon.shared.js#L76-L87

But that doesn't cover this hugeness. Perhaps there should be a `"Huge"` type there? cc @adamjspooner

Reply:
What would that size be used for? cc @cecileboucheron

Reply:
It didn't look like the design without that, IDK

Reply:
I changed it to 'Big':
![image](https://user-images.githubusercontent.com/862397/55506370-1b592700-560a-11e9-9154-bcd8aab955b1.png)

Reply:
@zanderz  Sorry, I meant to suggest adding `Huge` to Kb.Icon's `sizeType` prop, and use that to specify the size you were using originally, then this fontSize can just be `sizeType="Huge"` instead. But I guess I'm also OK with having the fontSize for now. We can always change it afterwards if we end up adding `Huge` to `Kb.Icon`.